### DISCUSSION TITLE
regex

### DISCUSSION THREAD

**Discussion by spani2431** [Chatterino/chatterino2#3131] [Q&A] [answered]
Ive been learning+writing regex to catch different slurs in order to help while modding but I'm unsure how to make highlights for the regex to work. Ive tested to make sure various versions, spellings, and "censoring" of the words work with the regex through https://regex101.com/ as the chatterino website suggests, but when I test to see if the highlight works in chatterino the text that should be detected is not highlighted. Ive tried to find answers elsewhere but havent found any help. Ive pasted the regex I want into the "pattern" section and enabled regex in the highlights tab but nothing happens. Thanks 👍

**Comment by apa420** [comment, marked as answer]
Here are some regex people use for that https://github.com/DatGuy1/common_badwords_twitch
The regex you posted is the same as the word without any fancy stuff https://regex101.com/r/W1nUCx/1

Make sure you haven't accidentally added a space somewhere where there shouldn't be any.

**Reply by spani2431** [reply]
oh wait mb thats not the one i wrote thanks tho
